.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+on:
+  pull_request:
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v3
+      with:
+        go-version: ^1.17 
```

Comment:
We should update this one to `1.18` (CLI will be updated by https://github.com/meroxa/cli/pull/362)